lib/portlayer/storage/nfs/vm.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nfs
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/vic/lib/config/executor"
+	"github.com/vmware/vic/lib/portlayer/exec"
+	"github.com/vmware/vic/lib/portlayer/storage"
+	"github.com/vmware/vic/pkg/trace"
+)
+
+const (
+	// TODO: pick potentially more sane defaults for these values
+	nfsMountOptions = "rw,noatime,vers=3,rsize=131072,wsize=131072,namlen=255,hard,proto=tcp,timeo=600,retrans=2,sec=sys,mountvers=3,mountproto=tcp,local_lock=none"
```

Comment:
Ah 😄 I was waiting for this comment. So yes `namelen` can go. 2 was the default for the `/bin/mount` if you do not specify we can try more. What would you suggest? `local_lock=none` is also a default for `/bin/mount` so this I will read up on and come back with an answer.